### PR TITLE
always return list from to_json

### PR DESCRIPTION
**Pull request by harunurhan**

### CODE REVIEW

File: tests/unit/test_utils.py [hunk 1]
```diff
@@ -118,3 +118,31 @@ def test_get_conf_set_for_key_path():
 def test_remove_prefix_value_error():
     with pytest.raises(ValueError):
         remove_prefix('a.b.c.d.e', 'a.b.c.e')
+
+
+def test_force_list_returns_empty_list_on_none():
```
Comment (drjova) [inline tests/unit/test_utils.py]:
```python
import pytest

@pytest.mark.parametrize(items, items_expected, [
('foo', ['foo']),
(['foo', 'bar'], ['foo', 'bar']),
(set(['foo', 'bar']), ['foo', 'bar']),
])
def test_force_list(items, items_expected):
    result = force_list(items)
    assert items_expected == result
```

☺️

**Review by drjova — APPROVED** [review]

File: json_merger/conflict.py [hunk 2]
```diff
@@ -121,9 +124,16 @@ def to_json(self):
...
-            'path': json_pointer,
-            'op': op,
-            'value': self.body,
-            '$type': self.conflict_type
-        })
+
+        conflict_values = force_list(self.body)
+        conflicts = []
+        for value in conflict_values:
+            if value is not None or self.conflict_type == 'REMOVE_FIELD':
```
Comment (ammirate) [inline json_merger/conflict.py:131]: Just for readability, maybe `if value` would be better

**Review by ammirate — APPROVED** [review]
Apart the `if value`, looks ok to me :+1:

File: json_merger/conflict.py (hunk 2 above)
Comment (harunurhan) [reply json_merger/conflict.py:131]: empty string, empty list, false would be skipped in that case.

**Pull request merged**